prov/tcp/src/tcpx_init.c

```diff
@@ -69,6 +70,12 @@ static void tcpx_init_env(void)
 	fi_param_define(&tcpx_prov,"port_high_range", FI_PARAM_INT,
 			"define port high range");
 
+	fi_param_define(&tcpx_prov,"tx_size", FI_PARAM_SIZE_T,
+			"define default tx context size");
```

Comment:
The above change means we can expand the text here to print the default.  E.g. fi_param_define(..., "define default tx context size (default: %zu)", tcpx_default_tx_size);

Reply:
Thanks for the suggestion, the new patch version has default values printed.